### ISSUE TITLE
Setting the window title

### ISSUE THREAD

**Issue by Lucky4Luuk** [Lokathor/beryllium#104]
When trying to use window.set_title(), it errors. I get the following error:
```
error[E0596]: cannot borrow data in a dereference of 'beryllium::window::gl_window::GlWindow' as mutable.
help: trait 'DerefMut' is required to modify through a dereference, but it is not implemented for 'beryllium::window::gl_window::GlWindow'```

I'm not doing any fancy, simply just creating a window and immediately trying to call `set_title` on it. Any idea what could cause this?

**Comment by Lokathor** [comment]
hmmmmmmmmm

**Comment by Lokathor** [comment]
Ooops, yeah my bad

**Comment by Lokathor** [comment]
fixed (by adding a DerefMut impl) in v0.2.1
